lib/wp2ghost.js

```diff
@@ -194,20 +196,48 @@ exports.fromStream = function(stream) {
       posts.push(post);
     });
 
-    xml.on('end', function() {
-      var ghost = {
-        "meta":{
-            "exported_on": exportDate.getTime() * 1000,
-            "version":"000"
-        },
-        "data":{
-            "posts": posts,
-            "tags": tags,
-            "posts_tags": posts_tags
-            //"users": users
-        }
-      };
-      resolve(ghost);
+    xml.on('end', async function () {
+      try {
+        // Fetch Ghost version
+        const version = await getGhostVersion();
+        const major = getMajorFromVersion(version);
+
+        // Use fallback if parsing fails
+        const ghostVersion = Number.isNaN(major) ? "6.0" : `${major}.0`;
+
+        const ghost = {
```

Comment:
There doesn't seem to be any actual difference in the exported data between the newer major version and the older one as far as I can tell? What's the actual difference in the export here?